Formula/jose.rb

```diff
@@ -0,0 +1,31 @@
+class Jose < Formula
+  desc "C-language implementation of Javascript Object Signing and Encryption"
+  homepage "https://github.com/latchset/jose"
+  url "https://github.com/latchset/jose/releases/download/v8/jose-8.tar.bz2"
+  sha256 "24e3d71e3da5a7913ab3c299381d76dfde488d91cb108b1a9527454bf1e9dc51"
+
+  head do
+    url "https://github.com/latchset/jose.git"
+
+    depends_on "libtool" => :build
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "jansson"
+  depends_on "lzlib"
+  depends_on "openssl"
+
+  def install
+    system "autoreconf", "-fiv" if build.head?
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"jose", "alg"
```

Comment:
In that case it would be fine to run "make check" within the `install` method if you'd like. Regardless, the test in the `test do` block needs to be something that exercises the installed software and verifies that it's working properly.